test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/Validation/DefaultObjectValidatorTests.cs

```diff
@@ -890,6 +918,39 @@ public void Validate_ForExcludedType_PropertiesMarkedAsSkipped()
             Assert.Empty(entry.Errors);
         }
 
+        [Fact]
+        public void Validate_ForExcludedCollectionType_PropertiesMarkedAsSkipped()
+        {
+            // Arrange
+            var validatorProvider = CreateValidatorProvider();
+            var actionContext = new ActionContext();
+            var modelState = actionContext.ModelState;
+            var validationState = new ValidationStateDictionary();
+
+            var validator = CreateValidator(typeof(List<string>));
+
+            var model = new List<string>()
+            {
+                "15",
+            };
+
+            // Note that user.ConfirmPassword has no entry in modelstate - we should not
+            // create one just to mark it as skipped.
+            modelState.SetModelValue("userIds[0]", "15", "15");
```

Comment:
Your :up::date: left this important, totally rad improvement out. :disappointed: